shard.yml

```diff
@@ -15,6 +15,6 @@ scripts:
 executables:
   - ameba
 
-crystal: ">= 1.4.0"
+crystal: "~> 1.4.0"
```

Comment:
Why do we need to exclude future major releases?

Reply:
That's the more conservative approach and one more in line with semver.

Reply:
I'm generally not a fan of upper bounded restrictions. But ameba uses internal compiler APIs which have no stability guarantee, so it might be good to be more conservative.
This means of course that every Crystal release needs a new ameba release. And that might not be the most practical solution, considering that breaking changes are actually still quite rare.

Reply:
Not _patch_ releases, just the _minor_/_major_ ones.